build/build-jemalloc-binaries.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Build cockroach binary using jemalloc as the allocator.
+
+set -euo pipefail
+
+source $(dirname $0)/build-common.sh
+
+# This is mildly tricky: This script runs itself recursively. The
```

Comment:
fair point. I was going to look into merging some of those, it is getting a bit unwieldy.